### PR TITLE
update theforeman npm packages to 4.14

### PR DESCRIPTION
**Pull request by MariaAga**
Matching foreman 2.3 packages.
vendor 4.0 doesn't have pf4, 4.14 has pf4 
* updated prettier to match foreman 
* Removed unused package raf

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,15 +22,15 @@
...
+    "@theforeman/vendor": ">= 4.14.0"
   },
   "devDependencies": {
     "@babel/core": "^7.7.0",
-    "@theforeman/builder": "^4.0.2",
-    "@theforeman/stories": "^4.0.2",
-    "@theforeman/test": "^4.0.2",
-    "@theforeman/vendor-dev": "^4.0.2",
-    "@theforeman/eslint-plugin-foreman": "^4.0.2",
+    "@theforeman/builder": "^4.14.0",
```
Comment (laviro) [inline package.json:29]: can you pin to that specific version of packaging? or using a Tilda, e.g: "~4.14.0"

**Review by laviro — COMMENTED** [review]
thanks for updating @MariaAga!
can you update also the rest of the packages according to what's in the packaging?

File: package.json (hunk 1 above)
Comment (MariaAga) [reply package.json:29]: For all the packages in the file?

Comment (laviro) [reply package.json]: Yeah if we already at it, I think it makes sense.

**Comment by MariaAga** [comment]
Updated prettier to foremans version and "locked" all package versions

File: package.json [hunk 2]
```diff
@@ -22,26 +22,26 @@
...
+    "@theforeman/builder": "~4.14.0",
+    "@theforeman/stories": "~4.14.0",
+    "@theforeman/test": "~4.14.0",
+    "@theforeman/vendor-dev": "~4.14.0",
+    "@theforeman/eslint-plugin-foreman": "~4.14.0",
+    "babel-eslint": "~10.0.0",
+    "eslint": "~6.7.2",
+    "eslint-plugin-spellcheck": "~0.0.17",
+    "prettier": "~1.19.1",
+    "raf": "~3.4.0",
```
Comment (laviro) [inline package.json:38]: Actually, this can be removed - didn't find any use for it on our repo :)

Comment (MariaAga) [reply package.json]: Removed :wastebasket:

File: package.json [hunk 3]
```diff
@@ -22,26 +22,25 @@
...
-    "@theforeman/eslint-plugin-foreman": "^4.0.2",
-    "babel-eslint": "^10.0.0",
-    "eslint": "^6.7.2",
-    "eslint-plugin-spellcheck": "^0.0.17",
-    "prettier": "^1.13.5",
-    "raf": "^3.4.0",
-    "stylelint": "^9.3.0",
-    "stylelint-config-standard": "^18.0.0",
-    "surge": "^0.20.3"
+    "@babel/core": "~7.7.0",
```
Comment (laviro) [inline package.json:28]:
```suggestion
    "@babel/core": "~7.7.4",
```

**Review by laviro — APPROVED** [review]
found a small version nitpick, LGTM

File: package.json (hunk 3 above)
Comment (MariaAga) [reply package.json:28]: It should still update the patch version when using `~` so I'm not sure it's needed

**Comment by laviro** [comment]
@MariaAga can you rebase?

**Comment by MariaAga** [comment]
rebased

**Comment by laviro** [comment]
Thanks @MariaAga !

**Pull request merged**